src/corelibs/U2View/src/ov_msa/find_pattern/FindPatternMsaWidget.cpp

```diff
@@ -969,6 +973,10 @@ void FindPatternMsaWidget::sl_findPatternTaskStateChanged() {
             prevPushButton->setEnabled(true);
             checkState();
             correctSearchInCombo();
+            if (setSelectionToFirstValuebleResult) {
+                setSelectionToFirstValuebleResult = true;
```

Comment:
It has no sence - you can get this line only if **setSelectionToFirstValuebleResult** is true.